### DISCUSSION TITLE
Unfortunately, we weren't able to approve your educational discount request for @eguner780.

### DISCUSSION THREAD

**Discussion by eguner780** [community/community#52586] [GitHub Education]
I've been trying to apply to the GitHub Student pack this whole week but everytime I get declined. When I upload a picture, it says that the image is manipulated and when I take a picture with my phone, it says this "Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved." Please, I would really appreciate it if a GitHub admin looked into this. I searched online and I've seen a bunch of people having the same problem. Please GitHub, fix this application system. It's clearly broken.
![image](https://user-images.githubusercontent.com/82319841/231360389-dc0f0e8c-a206-4586-8330-1cd213773620.png)

**Comment by IBRAHIMtaiyeroheemat** [comment]
I am facing the same problem

**Reply by sarimshoukat** [reply]
I've been facing the exact same issue

**Comment by samus-aran** [comment]
Hi y'all 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍

**Reply by illyrius666** [reply]
@samus-aran I for example am trying to upload a document that states my enrollment including: full name, date, student code, school signature, etc. I changed my GitHub name to my official name and wrote a short bio. I uploaded the document as a file, but that didn't work. so I tried making a direct photo of it via the browser, didn't work. It keeps giving me the following message:
"You are unlikely to be verified until you have completed your [GitHub user profile] with your full name as it appears in your academic affiliation document plus a short bio."
My school doesn't have instituted school emails. it also doesn't have school id cards. most universities in the Netherlands don't. So this whole progress is frustrating me a lot.
I would love someone to help me with this so I can make use of my student status.

Greetings.

**Comment by qa-apps** [comment]
I just trying to submit the documents but looks like the system is not working , it keep asking same documents over and over again.
